src/App.vue

```diff
@@ -615,7 +622,13 @@ export default {
 
 			// Breakout to breakout
 			return oldConversation.objectType === CONVERSATION.OBJECT_TYPE.BREAKOUT_ROOM && newConversation.objectType === CONVERSATION.OBJECT_TYPE.BREAKOUT_ROOM
-		}
+		},
+
+		showSignalingInternalWarning(token) {
+			if (this.$store.getters.isModerator || this.$store.getters.participantsList(token)?.length > 4) {
+				showWarning(t('spreed', 'Having a call with more than 4 participants without external signaling server can cause connectivity issues and cause high load on participating devices'))
```

Comment:
Instead of showWarning we should use a NcNoteCard in the header of the right sidebar below the Conversation name (above the lobby and other quick settings). Together with an actionable "Learn more" link which links to https://portal.nextcloud.com/article/Nextcloud-Talk/High-Performance-Backend/Installation-of-Nextcloud-Talk-High-Performance-Backend for now.